### DISCUSSION TITLE
GEO Map is not showing up

### DISCUSSION THREAD

**Discussion by PP-cnd** [wet-boew/wet-boew#9246] [Q&A] [answered]
https://wet-boew.github.io/v4.0-ci/demos/geomap/geomap-en.html
We use the GEO Map on our website and notice it doesn't show up now (and it used to be working just fine). We've just noticed that the map doesn't seem to be showing on the BOEW example webpage either. Any issues on this or workaround? Thank you.!

**Comment by jvanulde** [comment, marked as answer]
This is temporary. The map services are currently offline to address the Log4J vulnerability in the infrastructure.

**Reply by PP-cnd** [reply]
Make sense. Thank you for the update!

**Reply by jvanulde** [reply]
NP @PP-cnd the service endpoints are in the queue for scanning. Shouldn't be too long I hope.
